### ISSUE TITLE
PBot Explore: Collection Search Results

### ISSUE THREAD

**Issue by ecurrano** [paleobot/pbot-dev#287]
Here is a screenshot of what the "Quick Results" would look like for a collections search. We envisioned this in table format, but this may not be phone/tablet compatible. So, I could also see the information we want to show for each collection just listed underneath the collection name.

![Screen Shot 2024-03-07 at 9 57 42 AM](https://github.com/paleobot/pbot-dev/assets/72828382/d14d7746-cc4d-4304-98ca-fd1dbe2285a8)

For min and max ages: for standardization on this quick view results, converting everything to international epoch would be preferred. However, if that is too difficult, we could display whatever timescale was entered for min and max by the user.

When people click on a collection, they would see the full search results, and we love the way it is set up now. 

The Show me more data button is a placeholder to allow for data downloads.

**Comment by NoisyFlowers** [comment]
Some of the summary results is implemented in 
paleobot/pbot-client@11aeab475d9bb0daeb1236e32c813a3c7683cab0

merged to master and running on dev

**Comment by NoisyFlowers** [comment]
Location (lat/lon) search is in
paleobot/pbot-api@652479b8a184ba3ba1a6b66e56dc413c58c874a8
paleobot/pbot-client@
paleobot/pbot-client@
paleobot/pbot-client@

**Comment by NoisyFlowers** [comment]
Location (lat/lon) search is in
https://github.com/paleobot/pbot-api/commit/652479b8a184ba3ba1a6b66e56dc413c58c874a8
paleobot/pbot-client@1e362dc9dd9d57d4ac6bb18fbb2d0c4a304ccf71
paleobot/pbot-client@dc09931443d71efcf4a0979c95ff72235679552d

Merged to master and running on dev.

Note 1: To create the required location property in the Collection nodes, this cypher was run before turning this loose:
```
match (c:Collection  )  
set
    c.location = Point({latitude: c.lat, longitude: c.lon})
return c
```
Note 2: The api changes were made in a way that should maintain continuity with the previous. In other words, you can access the deprecated lat and lon properties as well as the new location.latitude/location.longitude The lat and lon properties accessed through the api are generated on the fly from those inside location.

Note 3: Ideally, we would also delete the old lat and lon properties in the nodes. I'm leaving them around for now in case there are any latent problems with the conversion cypher in note 1. However, they are no longer accessible through the api (see note 2).

**Comment by NoisyFlowers** [comment]
Note: 4: The search radius is currently set to 1500 meters. Let me know if we want it bigger/smaller.

**Comment by ecurrano** [comment]
Ah hah! I started playing about with what the radius was. We definitely want this bigger but Dori, Claire, Andrew, and I should consult on this. Maybe a whole degree? In v2, it might be nice to allow the user to specify a radius- I could see wanting a small radius for some questions but a big one for others. To some extent, you can get around the big radius now by doing a search by country or state.

**Comment by ecurrano** [comment]
Also, I think the quick search results look great. I'd prioritize getting other things done over standardizing min and max age or getting the number of specimens or OTUs at a site. Those would be icing on the cake.

**Comment by NoisyFlowers** [comment]
My comments regarding location search were meant for #283. I've moved them there for clarity. Ellen, you might want to move your relevant comment as well so we track the discussion there.

**Comment by ecurrano** [comment]
In the collection record page, under references: when multiple references, there is nothing separating the different references, not even a space. Possible to have each reference as separate lines, like for specimens?

**Comment by ecurrano** [comment]
This may be too much work for the amount of time we have left, but it would be very helpful to add an accordion for OTUs to be able to show all the OTUs present at the site.

**Comment by doricon** [comment]
@ecurrano, I agree - I like that there is a list of linkable specimens, but a list of the OTUs identified at the site would be really useful as well. 

I thought we could get that info a different way by searching OTUs and just refine by collection, but I just tried and realized that you can't search by a specific collection, just location/time.

**Comment by aazaff** [comment]
There's no way to search by collection id? That's bad.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: doricon ***@***.***>
Sent: Wednesday, September 25, 2024 9:23:09 AM
To: paleobot/pbot-dev ***@***.***>
Cc: Zaffos, Andrew - (azaffos) ***@***.***>; Assign ***@***.***>
Subject: [EXT] Re: [paleobot/pbot-dev] PBot Explore: Collection Search Results (Issue #287)


External Email

________________________________
@ecurrano<https://github.com/ecurrano>, I agree - I like that there is a list of linkable specimens, but a list of the OTUs identified at the site would be really useful as well.

I thought we could get that info a different way by searching OTUs and just refine by collection, but I just tried and realized that you can't search by a specific collection, just location/time.
